README.mdtmpl

```diff
@@ -1,75 +0,0 @@
-# sass_builder
```

Comment:
I know this file was used to generate the readme and ensure that the code examples in the readme match the code from the example directory. I don't know what tool exactly was being used to generate the README.md file. 

@luisvt Do you remember what tool you used?